pkg/scalers/arangodb_scaler.go

```diff
@@ -31,37 +30,37 @@ type dbResult struct {
 
 // arangoDBMetadata specify arangoDB scaler params.
 type arangoDBMetadata struct {
-	// Specify arangoDB server endpoint URL or comma separated URL endpoints of all the coordinators.
+	// Specify arangoDB server endpoint URL or comma separated URL Endpoints of all the coordinators.
 	// +required
-	endpoints string
+	Endpoints string `keda:"name=endpoints, order=authParams;triggerMetadata"`
 	// Authentication parameters for connecting to the database
 	// +required
-	arangoDBAuth *authentication.AuthMeta
+	arangoDBAuth *authentication.AuthMeta // TODO: Fix this
```

Comment:
I think it should be sufficient to change this to `*authentication.Config`, let me know if that works or if there is something still missing
https://github.com/kedacore/keda/blob/a375d1768a8add922e65d55968515eea12b86413/pkg/scalers/prometheus_scaler.go#L55
if there is some additional logic requiring the `authentication.AuthMeta`, you can use this convertor
https://github.com/kedacore/keda/blob/a375d1768a8add922e65d55968515eea12b86413/pkg/scalers/authentication/authentication_types.go#L156-L157

Reply:
Yes  saw that you have marked this struct as deprecated in the previous PR so I put a note there so that I could ask you some questions about it :smile: